build.gradle

```diff
@@ -50,3 +54,7 @@ wrapper {
   gradleVersion = "5.6.4"
   distributionType = Wrapper.DistributionType.BIN
 }
+
+compileGroovy.dependsOn.remove('compileJava')
+compileKotlin.dependsOn compileGroovy
```

Comment:
I wonder if this is fixed by  using a more recent version of KGP? 🤔

Reply:
Gradle 6.1 gives a more regular way (see shared thread above)

Reply:
Thanks